common/plugins/org.jboss.tools.common.core/src/org/jboss/tools/common/util/BeanUtil.java

```diff
@@ -74,5 +75,20 @@ public static String getPropertyName(String methodName) {
 		}
 		return null;
 	}
+	
+	public static String getDefaultBeanName(String className){
+		int lastDotPosition = className.lastIndexOf("."); //$NON-NLS-1$
+		if(lastDotPosition >= 0 && className.length() > lastDotPosition){
+			className = className.substring(+1);
```

Comment:
I guess you meant className = className.substring(lastDotPosition + 1);
Not just "+ 1".

And we should add a unit test for this method.